src/main/fractions.ts

```diff
@@ -23,7 +23,7 @@ export class FractionsConstraint implements ValidatorConstraintInterface {
       throw new Error('Maximum allowed decimal places has to be specified and positive value')
     }
 
-    if (value === undefined) {
+    if (value === undefined || value === null) {
```

Comment:
why would a value be null? 
We don't use null in our codebase

Reply:
cmc-legal-frontend has a unit test where a null amount should be accepted. This validator was causing that test to fail where it previously passed (Line 58 here is the test https://github.com/hmcts/cmc-legal-rep-frontend/blob/master/src/test/app/forms/models/amount.ts). 

Would it be worth changing the codebase on the legal frontend to ensure null isn't being used and change the test accordingly, or keep it as is?

Reply:
change the test, some people have coded against it despite us guiding them not too, we should add a lint rule to prevent it really.

There has been approx 2 cases where we've had to use null but thats out of 100s-1000s of checks